components/authn-service/server/server.go

```diff
@@ -24,6 +25,10 @@ import (
 	"github.com/chef/automate/lib/version"
 )
 
+const (
+	IngestAccessPolicyID = "ingest-access"
```

Comment:
Let's be consistent with authz-service:
```suggestion
	IngestPolicyID = "ingest-access"
```